packages/rich-text/src/plugins/Quote/index.tsx

```diff
@@ -136,3 +147,19 @@ export function Quote(props: Slate.RenderLeafProps) {
     </blockquote>
   );
 }
+
+export function createQuotePlugin(): SlatePlugin {
+  return {
+    pluginKeys: BLOCKS.QUOTE,
+    renderElement: getRenderElement(BLOCKS.QUOTE),
+    onKeyDown: withQuoteEvents,
+  };
+}
+
+export const withQuoteOptions: CustomSlatePluginOptions = {
+  [BLOCKS.QUOTE]: {
+    type: BLOCKS.QUOTE,
+    component: Quote,
+    hotkey: ['mod+shift+1'],
```

Comment:
I think this won't trigger anything because we are not using their toggle functions on line 155, we can safely remove it.